### DISCUSSION TITLE
Hi 🖖

### DISCUSSION THREAD

**Discussion by pheeria** [Lernwerkstatt/site#431] [General]
How are you doing? 🙂

**Comment by danielstreif** [comment]
Good 🙂 Any new features to work on?

**Reply by pheeria** [reply]
It's sad that @cappyccino doesn't want to support us. Previously, we had at least some cookies 😄

**Reply by pheeria** [reply]
We have some failing e2e tests, that would be nice to fix them 🙂 
Btw, we managed to persuade FB to give us read permissions. Do you have a FB account?
I'll add you as an admin.

**Reply by danielstreif** [reply]
Sure, I can have a look at the tests 🙂 
I technically do have an account with FB but I think the last time I logged in has been two years ago 😅 Probably doesn't make much sense to add me.

**Reply by danielstreif** [reply]
And once possible we should definitely meet for cookies again! 🍪
